packages/@aws-cdk/aws-appsync/test/appsync-schema.test.ts

```diff
@@ -32,6 +33,16 @@ beforeEach(() => {
   stack = new cdk.Stack();
 });
 
+test('appsync fails when both file path and s3 location are configured', () => {
```

Comment:
I would say the schema fails creation, "appsync fails" is kind of a weird statement 😃.